pages/api/volunteer.tsx

```diff
@@ -0,0 +1,16 @@
+import { supabase } from '../../components/utils/supabaseClient'
```

Comment:
you're not using react here, so `*.ts` would be what you would use for next API routes.

Reply:
It took me a few reads to understand the comment here, so for the next person reading it, I think @mitchelvanbever is saying the file name should be `volunteer.ts` instead of `volunteers.tsx`.

Reply:
That's correct

Reply:
Good call! Thanks for elaboration Mike